lib/screens/NewPatientScreen.dart

```diff
@@ -573,6 +608,39 @@ class _NewPatientFormState extends State<_NewPatientForm> {
   // OTHER
   // ----------
 
+  /// Removes all non-number characters and inserts dashes to make number more
+  /// readable. E.g. 12345678 becomes 12-345-678.
+  ///
+  /// Does not trim the number and only inserts two dashes. So if you pass it a
+  /// long number string, the number will stay long. E.g. 1234567890123 becomes
+  /// 12-345-67890123.
+  ///
+  /// If a [countryCode] is passed it will be prefixed with a dash. E.g.
+  /// countryCode='266' returns +266-12-345-678.
+  String _formatPhoneNumber(String phoneNumber, {String countryCode}) {
+    String onlyNumbers = phoneNumber.replaceAll(RegExp('[^0-9]'), '');
+    String formattedNumber;
+    if (onlyNumbers.length >= 2) {
+      formattedNumber = onlyNumbers.substring(0, 2) + '-' + onlyNumbers.substring(2, onlyNumbers.length);
+    }
+    if (onlyNumbers.length >= 5) {
+      formattedNumber = onlyNumbers.substring(0, 2) + '-' + onlyNumbers.substring(2, 5) + '-' + onlyNumbers.substring(5, onlyNumbers.length);
+    }
+    if (countryCode != null && countryCode.isNotEmpty) {
+      String countryCodeOnlyNumbers = countryCode.replaceAll(RegExp('[^0-9]'), '');
+      formattedNumber = '+$countryCodeOnlyNumbers-$formattedNumber';
+    }
+    return formattedNumber;
+  }
+
+  /// Removes all non-number characters and inserts a prefix 'PEBRA_'
+  /// E.g. '123' becomes 'PEBRA_123'.
+  String _formatStickerNumber(String stickerNumber) {
+    String onlyNumbers = stickerNumber.replaceAll(RegExp('[^0-9]'), '');
+
+    return 'PEBRA_' + onlyNumbers;
```

Comment:
Use "P" as a prefix instead of "PEBRA_" (the requirement just changed today, see update in #14).